bin/custom-properties-spell-check.js

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
+
+require('../scripts/custom-properties/spell-check');
```

Comment:
Did we change the read-write privs on the bin files `pack.js`, `serve-static.js`, etc? I notice its changed from `100644` to `100755`

Reply:
Yeah, not sure why these changed, need to revert the read-write privs on these files.